src/modules/ekf2/EKF/aid_sources/external_vision/ev_vel_control.cpp

```diff
@@ -232,10 +243,86 @@ void Ekf::controlEvVelFusion(const extVisionSample &ev_sample, const bool common
 	}
 }
 
+bool Ekf::fuseEvVelocity(estimator_aid_source3d_s &aid_src, const extVisionSample &ev_sample)
+{
+	if (ev_sample.vel_frame == VelocityFrame::BODY_FRAME_FRD) {
+
+		VectorState H;
+		estimator_aid_source1d_s current_aid_src;
+
+		for (uint8_t index = 0; index <= 2; index++) {
+			current_aid_src.timestamp_sample = aid_src.timestamp_sample;
+
+			if (index == 0) {
+				sym::ComputeEvBodyVelHx(_state.vector(), &H);
+
+			} else if (index == 1) {
+				sym::ComputeEvBodyVelHy(_state.vector(), &H);
```

Comment:
Shouldn't this be the original state vector before any EV fusion?

Mag fusion for example. https://github.com/PX4/PX4-Autopilot/blob/b3ff409d27800cf2462ef8285a3afde72f6146c4/src/modules/ekf2/EKF/aid_sources/magnetometer/mag_fusion.cpp#L71-L75